cmd/noms/noms.go

```diff
@@ -44,49 +29,70 @@ var actions = []string{
 	"nomming on",
 }
 
-func usageString() string {
-	i := rand.New(rand.NewSource(time.Now().UnixNano())).Intn(len(actions))
-	return fmt.Sprintf(`Noms is a tool for %s Noms data.`, actions[i])
+// CommandHandler - a callback passed to commands
```

Comment:
Fair enough! vet complains at me when I don't put comments on public methods, so sometimes I feel like my go comments are redundant because the name is descriptive enough.